sbom/formatted_reader.go

```diff
@@ -133,7 +133,7 @@ func (f *FormattedReader) Read(b []byte) (int, error) {
 				spdxOutput["documentNamespace"] = uri.String()
 			}
 
-			output, err = json.Marshal(spdxOutput)
+			output, err = json.MarshalIndent(spdxOutput, "", "\t")
```

Comment:
nit:
I see that in syft's SBOM formatting implementation, for SPDX json, they [use](https://github.com/anchore/syft/blob/1344889766743beb736aafdfb29266910b738fbb/internal/formats/spdx22json/encoder.go#L16) 
```go
	enc := json.NewEncoder(output)
	// prevent > and < from being escaped in the payload
	enc.SetEscapeHTML(false)
	enc.SetIndent("", " ")
```
and for CycloneDX JSON, they [use the cyclonedx-go package](https://github.com/anchore/syft/blob/1344889766743beb736aafdfb29266910b738fbb/internal/formats/cyclonedxjson/encoder.go#L13), which [uses](https://github.com/CycloneDX/cyclonedx-go/blob/429d353cfcdbfedf367f597cbdde2a840ebf29df/encode.go#L44)
```go
	if j.pretty {
		encoder.SetIndent("", "  ")
	}
```


So, with this indent, we're effectively replacing spaces in the SBOM files with tabs. For consistency, I think it makes sense to use spaces instead. It's a bit weird that one JSON format uses single-space indentation and the other uses two. I figure we just pick one and stick with it.

Reply:
@fg-j Good idea - can you clarify - you want us to use spaces instead but just pick either one space or two for all formats? What would stop us from just using one space for SPDX and two for CycloneDx so we match their underlying representations?

Reply:
I have opted for using one space for SPDX and two for Cyclonedx so that our output SBOMs just match the upstream as much as possible.

Reply:
Fair point. We may as well match the existing implementations since we're already calling `json.MarshalIndent` separately for SPDX and CycloneDX.